include/rocksdb/convenience.h

```diff
@@ -330,6 +330,8 @@ void CancelAllBackgroundWork(DB* db, bool wait = false);
 // Snapshots before the delete might not see the data in the given range.
 Status DeleteFilesInRange(DB* db, ColumnFamilyHandle* column_family,
                           const Slice* begin, const Slice* end);
+Status DeleteFilesInRange(DB* db, ColumnFamilyHandle* column_family,
```

Comment:
can we call the new API `DeleteFilesInRanges`?

Reply:
Sounds good.